lesson24_cp1/project_users/views.py

```diff
@@ -0,0 +1,72 @@
+import hashlib
+from django.db.utils import IntegrityError
+from django.shortcuts import render
+from django.views import View
+from project_users.models import CustomUser
+
+
+# Create your views here.
+
+class HomePageView(View):
+    TITLE = "Home Page"
+
+    def get(self, request, *args, **kwargs):
+        count = CustomUser.objects.count()
+        data = CustomUser.objects.filter(pk__gte=count - 10)
+        new_data = []
+        for user in data:
+            before_at, after_at = user.email.split('@')
+            hidden_part = '*' * (len(before_at) - 2)
+            new_data.append(f'{before_at[:2:]}{hidden_part}@{after_at}')
```

Comment:
we dont need write `:` after `end` param using slice.
unless we want define a `step` param